gradle/libs.versions.toml

```diff
@@ -12,9 +12,8 @@ moshi = "1.15.0"
 minSdk = "25"
 compileSdk = "33"
 
-# Maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/5e972ea
-layoutlib = "2022.3.1-5e972ea"
-layoutlibPrebuiltSha = "5e972ea"
+# Maps to this tag: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/refs/tags/studio-2022.3.1
+layoutlib = "2022.3.1"
```

Comment:
This needs a single re-publish without the SHA hash, from then on this is the only number that needs updating.

Reply:
The builds will fail until this is published, that's why CI is all red.